src/components/responsive-flyout/styles/OverlayHeader.scss

```diff
@@ -0,0 +1,18 @@
+@import '../../../styles/variables';
+@import './variables';
+
+.overlay-header {
+    position: relative;
+    z-index: 1;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    height: $overlay-header-height;
+    padding: 13px 16px;
+    background-color: white;
+    box-shadow: 0 2px 4px 0 rgba(0, 0, 0, .08);
+
+    .oh-content {
```

Comment:
I didn't know what this was when I read it in `OverlayHeader`. Some other Boxers could be as dull as I am on their worst days 😄  On our behalf, would it make sense to go with a more explicit name, such as `overlay-header-content`?

Reply:
Changed it to bdl-OverlayHeader